recipes/colcon-core/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "colcon-core" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/colcon/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 70165765d80234352a5ceb05b28b164be6b964303e5b59f9612911987f4ee438
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  entry_points:
+    - colcon = colcon_core.command:main
+
+requirements:
+  host:
+    - pip
+    - python {{ python }}
+  run:
+    - python >=3.5
```

Comment:
`conda_build_config.yaml` shouldn't be used at all. The issue is that you only changed the `host` requirement. The `run` requirement should be changed to match.

The pattern is either:
```yaml
build:
  noarch: python

requirements:
  host:
    - python >=3.5
  run:
    - python >=3.5
    - dependency_that_is_for_all_platforms_and_versions
```
or:
```yaml
build:
  skip: true  # [py<35]

requirements:
  host:
    - python >=3.5
  run:
    - python >=3.5
    - some_windows_only_dependency  # [win]
```

Reply:
As long as I am specifying the `python >=3.5` under `host`. The renaming error will manifest.

It seems only the below combination works for me locally. :(
(Trigger another CI build to confirm other platforms...)

```yaml
build:
  skip: true  # [py<35]

requirements:
  host:
    - python
  run:
    - python >=3.5
    - some_windows_only_dependency  # [win]
```

Reply:
Sorry there was a mistake in the second code block. I've corrected it now.

Reply:
No problem! And finally CI shows green again. :)